tsconfig.json

```diff
@@ -6,7 +6,7 @@
 		"preserveConstEnums": true,
 		"strictNullChecks": true,
 		"sourceMap": true,
-		"target": "es5",
+		"target": "es6",
```

Comment:
Is there a reason we don't want to support <ES6?

Reply:
I get loads of Iterable not found errors in Bluebird when I target es5